pkg/grpc/server.go

```diff
@@ -57,6 +56,7 @@ func NewAPIServer(api API, config ServerConfig, tracingSpec config.TracingSpec)
 		config:      config,
 		tracingSpec: tracingSpec,
 		kind:        apiServer,
+		logger:      logger.NewLogger("dapr.runtime.grpc.api"),
```

Comment:
Have you checked the log? can you see correct app_id? I remember that NewLogger needs to be initialized before [this code](https://github.com/dapr/dapr/blob/891adb81053d354d7afd0fb91c3029edd1ef502d/pkg/runtime/cli.go#L61).

Reply:
You are correct the `app_id` field is indeed missing from the log lines from the `dapr.runtime.grpc.*` loggers. I'll add a fix.

```
time="2020-04-10T09:08:05.814018253Z" level=info msg="starting Dapr Runtime -- version edge -- commit v0.6.0-8-g345c104" app_id=nodeapp instance=nodeapp-554d49c67c-b4pxk scope=dapr.runtime
 type=log ver=edge
...

time="2020-04-10T09:08:05.935904758Z" level=info msg="enabled metrics grpc middleware" instance=nodeapp-554d49c67c-b4pxk scope=dapr.runtime.grpc.api type=log ver=edge

time="2020-04-10T09:08:05.935960158Z" level=info msg="API gRPC server is running on port 50001" app_id=nodeapp instance=nodeapp-554d49c67c-b4pxk scope=dapr.runtime type=log ver=edge

time="2020-04-10T09:08:05.936001658Z" level=info msg="enabled metrics grpc middleware" instance=nodeapp-554d49c67c-b4pxk scope=dapr.runtime.grpc.internal type=log ver=edge

time="2020-04-10T09:08:05.936035458Z" level=info msg="sending workload csr request to sentry" instance=nodeapp-554d49c67c-b4pxk scope=dapr.runtime.grpc.internal type=log ver=edge
time="2020-04-10T09:08:05.984279901Z" level=info msg="certificate signed successfully" instance=nodeapp-554d49c67c-b4pxk scope=dapr.runtime.grpc.internal type=log ver=edge
time="2020-04-10T09:08:05.984437501Z" level=info msg="internal gRPC server is running on port 50002" app_id=nodeapp instance=nodeapp-554d49c67c-b4pxk scope=dapr.runtime type=log ver=edge
time="2020-04-10T09:08:05.993664471Z" level=info msg="starting workload cert expiry watcher. current cert expires on: 2020-04-11 09:23:05 +0000 UTC" instance=nodeapp-554d49c67c-b4pxk scope
=dapr.runtime.grpc.internal type=log ver=edge
time="2020-04-10T09:08:05.994440269Z" level=info msg="enabled cors http middleware" app_id=nodeapp instance=nodeapp-554d49c67c-b4pxk scope=dapr.runtime.http type=log ver=edge
...

Reply:
Latest commit should fix this...

```
time="2020-04-10T09:21:43.811327513Z" level=info msg="enabled metrics grpc middleware" app_id=nodeapp instance=nodeapp-554d49c67c-92xb4 scope=dapr.runtime.grpc.api type=log ver=edge
time="2020-04-10T09:21:43.811426512Z" level=info msg="API gRPC server is running on port 50001" app_id=nodeapp instance=nodeapp-554d49c67c-92xb4 scope=dapr.runtime type=log ver=edge
time="2020-04-10T09:21:43.811465212Z" level=info msg="enabled metrics grpc middleware" app_id=nodeapp instance=nodeapp-554d49c67c-92xb4 scope=dapr.runtime.grpc.internal type=log ver=edge
```